apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -292,10 +292,24 @@ function initializeBlocklyWrapper(blocklyInstance) {
       container.appendChild(svg);
       svg.appendChild(workspace.createDom());
       Blockly.Xml.domToBlockSpace(workspace, xml);
+
+      // Loop through all the child blocks and remove transform
+      const blocksInWorkspace = workspace.getAllBlocks();
+      blocksInWorkspace.forEach(block => {
+        block.svgGroup_.removeAttribute('transform');
+      });
+
       // Shrink SVG to size of the block
       const bbox = svg.getBBox();
       svg.setAttribute('height', bbox.height + bbox.y);
       svg.setAttribute('width', bbox.width + bbox.x);
+      // Add a transform to center read-only blocks on their line
+      const notchHeight = workspace.getRenderer().getConstants().NOTCH_HEIGHT;
+
+      svg.setAttribute(
+        'style',
+        `transform: translate(0px, ${notchHeight + 7}px)`
```

Comment:
The block has a height of 32 pixels and the line height of the text is 25, which is how I get this 7, but I'm not sure if there's a better way to represent this that isn't a 'magical number'.

Reply:
can you declare the 7 as a `const` variable doing something like `const blockPadding = lineHeight - textHeight` to make it more clear in the code where it came from?

Reply:
I couldn't find a way to do this through calculation, but I broke it out into a const with a comment, so hopefully that makes sense.